esrally/__init__.py

```diff
@@ -17,6 +17,7 @@
 
 import os
 import sys
+import urllib
```

Comment:
Beyond the scope of this PR, but should we slowly settle on having alphabetically sorted imports?

Reply:
I *try* to keep imports somewhat logically grouped so far but I don't mind too much. I am just not certain how we would enforce this on an existing code base. But let's discuss this separately.